install/ci-vm/ci-linux/startup-script.sh

```diff
@@ -9,24 +9,28 @@ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BF
 echo "deb https://download.mono-project.com/repo/ubuntu stable-focal main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list
 sudo apt update
 apt install mono-complete -y
+apt-get install -y libtesseract-dev
```

Comment:
Is it possible to combine the installations of L11 & L12?

Reply:
Yes, we can, I'll update the PR.